src/libs/actions/IOU.js

```diff
@@ -1512,7 +1512,7 @@ function deleteMoneyRequest(transactionID, reportAction, isSingleTransactionView
         }
 
         updatedIOUReport.lastMessageText = iouReportLastMessageText;
-        updatedIOUReport.lastVisibleActionCreated = lastVisibleAction.created;
+        updatedIOUReport.lastVisibleActionCreated = lodashGet(lastVisibleAction, 'created');
```

Comment:
Why you are changing it to lodash?

Reply:
@kubabutkiewicz  It was because of a change I made to `getLastVisibleAction` to stop returning `{}` when it should be returning `null`, and then I addressed the changes throughout the code to avoid regressions.